backends-common/redis/src/test/java/org/apache/james/backends/redis/RedisConfigurationTest.scala

```diff
@@ -48,6 +49,7 @@ class RedisConfigurationTest extends AnyFlatSpec with Matchers {
     config.setListDelimiterHandler(new DefaultListDelimiterHandler(','))
     config.addProperty("redisURL", "redis://localhost:6379,redis://localhost:6380")
     config.addProperty("redis.topology", "cluster")
+    config.addProperty("redis.readFrom", "any")
```

Comment:
Can we have a test that `readFrom=invalid` ?

Reply:
done